firecrown/cosmosis/run.py

```diff
@@ -62,7 +52,7 @@ def run_cosmosis(config, data):
         pool.close()
 
 
-def _make_parallel_pool(cosmosis_config):
+def _make_parallel_pool(config):
```

Comment:
We need to update doc strings for changes in inputs

Reply:
done